lab-peter/lib/readfile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
+
+var myArray = [];
+
+module.exports = function(callback) {
+  fs.readFile(`${__dirname}/../data/one.txt`, function(err, data) {
+    if (err) throw new Error('text one is wrong');
```

Comment:
Nice error handling! Although as you heard today, we won't be throwing errors anymore. But it's good that you're thinking about error handling! :)